src/index.js

```diff
@@ -82,11 +82,21 @@ const permissionsMap = {
  * @param {*} result
  */
 
+/**
+ * @typedef {string} TeamName
+ * @typedef {Array<string>} TeamMembers
+ * @typedef {Object<TeamName, TeamMembers>} Team
```

Comment:
Couldn't this just be

```suggestion
 * @typedef {Object<string, Array<string>>} Team
```